src/main/webapp/cdn/blockly/generators/propc/base.js

```diff
@@ -480,3 +590,388 @@ Blockly.propc.compare_colors = function() {
     var code = 'compareRRGGBB(' + color1 + ', ' + color2 + ')';
     return [code];
 };
+
+Blockly.Blocks.logic_compare = {
+    // Comparison operator.
+    category: Blockly.LANG_CATEGORY_LOGIC,
+    init: function() {
+        if(profile.default.description === "Scribbler Robot") {
```

Comment:
@MatzElectronics I'm making a note here so we can come back and work on this. Ideally, the code that sets the profile.default.description should use a string constant. That same constant would then be referenced in this if() statement - and others that are sure to exist - so that we are not going to break anything if the text 'Scribbler Robot' is changed in the future.